lib/class-wp-rest-menu-items-controller.php

```diff
@@ -516,135 +654,158 @@ public function get_item_schema() {
 			'description' => __( 'Unique identifier for the object.' ),
 			'type'        => 'integer',
 			'default'     => 0,
+			'minimum'     => 0,
 			'context'     => array( 'view', 'edit', 'embed' ),
 			'readonly'    => true,
 		);
 
-		$schema['properties']['menu_id'] = array(
-			'description' => __( 'Unique identifier for the menu.' ),
-			'type'        => 'integer',
-			'context'     => array( 'edit' ),
-			'default'     => 0,
-		);
-
 		$schema['properties']['type_label'] = array(
 			'description' => __( 'Name of type.' ),
 			'type'        => 'string',
-			'context'     => array( 'view', 'embed' ),
+			'context'     => array( 'view', 'edit', 'embed' ),
 			'readonly'    => true,
 		);
 
 		$schema['properties']['type'] = array(
-			'description' => __( 'Type of menu item' ),
+			'description' => __( 'The family of objects originally represented, such as "post_type" or "taxonomy".' ),
 			'type'        => 'string',
 			'enum'        => array( 'taxonomy', 'post_type', 'post_type_archive', 'custom' ),
 			'context'     => array( 'view', 'edit', 'embed' ),
+			'default'     => 'custom',
+			'arg_options' => array(
+				'sanitize_callback' => 'sanitize_key',
```

Comment:
Why is this necessary in this case? Shouldn't `enum` cover it?

Reply:
It maybe over kill here. Should I remove it

Reply:
I think so.

Reply:
+1, should be removed